live/prod/data-stores/redis/main.tf

```diff
@@ -24,7 +24,7 @@ module "redis" {
   stage  = "prod"
   name   = "redis"
 
-  alarm_cpu_threshold_percent  = 90
+  alarm_cpu_threshold_percent  = 85
```

Comment:
Why we change this? And how you decide that number?

Reply:
we had this set to 85% on other resources, I wanted to keep the same number for the alerts. What this does if the cpu of the redis cluster goes above 85% for more than 3 minutes it will notify us on Slack